sdk/metric/config.go

```diff
@@ -43,3 +47,13 @@ type resourceOption struct {
 func (o resourceOption) Apply(config *Config) {
 	config.Resource = o.Resource
 }
+
+func WithMetricsProcessors(processors []MetricsProcessor) Option {
+	return metricsProcessorsOption(processors)
+}
+
+type metricsProcessorsOption []MetricsProcessor
+
+func (p metricsProcessorsOption) Apply(config *Config) {
+	config.MetricsProcessors = p
```

Comment:
```suggestion
	config.MetricsProcessors = append(config.MetricsProcessors, p...)
```

Perhaps append to the list to allow multiple independent additions of metrics processors?